### PR TITLE
fix(editor): throttle render linked doc card when blockUpdated

### PR DESCRIPTION
**Pull request by graydawnc**
[BS-2223](https://linear.app/affine-design/issue/BS-2223/性能问题：linked-doc-card-view加载西藏的文档，app-直接挂了)

### CODE REVIEW

**Comment by graydawnc** [comment]
* **#9361** <a href="https://app.graphite.dev/github/pr/toeverything/AFFiNE/9361?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈 <a href="https://app.graphite.dev/github/pr/toeverything/AFFiNE/9361?utm_source=stack-comment-view-in-graphite" target="_blank">(View in Graphite)</a>
* `canary`
<hr />

## How to use the Graphite Merge Queue

Add either label to this PR to merge it via the merge queue:
- <kbd>merge</kbd> - adds this PR to the back of the merge queue
- <kbd>hotfix</kbd> - for urgent hot fixes, skip the queue and merge this PR next

You must have a Graphite account in order to use the merge queue. Sign up using [this link](https://app.graphite.dev/invite/github/toeverything?ref=merge-queue-instructions-in-stack-comment&prId=9361).

<h2></h2>
<sub>

An organization admin has enabled the [Graphite Merge Queue](https://graphite.dev/docs/graphite-merge-queue) in this repository.

</sub>

<sub>

Please do not merge from GitHub as this will restart CI on PRs being processed by the merge queue.

</sub>

<h2></h2>

This stack of pull requests is managed by <a href="https://graphite.dev?utm-source=stack-comment"><b>Graphite</b></a>. Learn more about <a href="https://stacking.dev/?utm_source=stack-comment">stacking</a>.

**Review by doodlewind — APPROVED** [review]

File: blocksuite/affine/block-embed/src/embed-linked-doc-block/embed-linked-doc-block.ts [hunk 1]
```diff
@@ -301,24 +301,28 @@ export class EmbedLinkedDocBlockComponent extends EmbedBlockComponent<EmbedLinke
...
+
+            if (payload.type === 'add' && payload.init) {
+              return;
+            }
+
+            this._load().catch(e => {
+              console.error(e);
+              this.isError = true;
+            });
+          }, 60)
```
Comment (Saul-Mirone) [inline blocksuite/affine/block-embed/src/embed-linked-doc-block/embed-linked-doc-block.ts]: put it into a const, and explain why its 60

**Review by Saul-Mirone — CHANGES_REQUESTED** [review]
let's do some improvements

**Review by Saul-Mirone — APPROVED** [review]
LGTM

**Pull request merged**